backend/main/models/community.go

```diff
@@ -217,8 +218,9 @@ func (c *Community) UpdateCommunity(db *s.Database, p *UpdateCommunityRequestPay
 	contract_name = COALESCE($16, contract_name),
 	contract_addr = COALESCE($17, contract_addr),
 	public_path = COALESCE($18, public_path),
-	threshold = COALESCE($19, threshold)
-	WHERE id = $20
+	threshold = COALESCE($19, threshold),
```

Comment:
to avoid confusion can we remove `threshold` ? it's been used for the creation and update requests

Reply:
Good idea - I was actually looking at the DB in another ticket and was getting confused about that column.

Reply:
Done.

Reply:
We can do this in another ticket to clean up once we confirm that this value isn't used elsewhere unexpectedly.